packages/core/src/builder.class.ts

```diff
@@ -2227,6 +2227,11 @@ export class Builder {
     let instance: Builder = this;
     let finalLocale =
       options.locale || options.userAttributes?.locale || this.getUserAttributes().locale;
+
+    if (!('noTraverse' in options)) {
```

Comment:
should it be?
```suggestion
    if (!('noTraverse' in options) && this.apiEndpoint === 'content') {
```

Reply:
When will there be any `apiEndpoint` which will not be `content`?

Reply:
@samijaber Replied to your question in slack thread.

Reply:
makes sense